### ISSUE TITLE
Icon missing for gnome screen keyboard

### ISSUE THREAD

**Issue by Ninlives** [SylEleuth/gruvbox-plus-icon-pack#1]
In gnome, after applying this icon pack the screen keyboard looks like this:
![image](https://user-images.githubusercontent.com/17873203/226167051-f6714d4d-ae66-45ae-a1f1-bcc37d587fcd.png)

This is what screen keyboard looks like using the default Adwaita icon pack:
![image](https://user-images.githubusercontent.com/17873203/226167083-b7be735f-3dc3-4afc-870c-5828741d40bf.png)

**Comment by SylEleuth** [comment]
Thank You for letting me know. It will take some time because I would have to install Gnome on VM. I'm not familiar with that DM. Is that screen keyboard an extension, or built-in Gnome functionality?

**Comment by SylEleuth** [comment]
I found a culprit. Two actually - apps/keyboard-symbolic.svg and devices/keyboard-symbolic.svg. I removed them so there is no need to remove any others having "keyboard" in their name. I've also added shift, enter and keyboard-hide icons. Unfortunately I couldn't find a name for the language change icon. That is why I removed 2 mentioned above. Without them language icon will fallback to default for on-screen keyboard. Which btw must be internal because I couldn't find in Adwaita pack anything similar to that clog (nor shift, enter or hide).
